### PR TITLE
Fix a bug of module string analyzing, style improvement of small card and Header

### PR DESCRIPTION
**Pull request by SheepChef**
## Changes

Fix a bug of module string analyzing. (lines[ ] was not declared)
Formatting Code using prettier.

## Why 

Bug fixes.

## Checkmarks

[ ] The modified functions have been tested.
[ ] Used the same indentation as the rest of the project.
[ ] Updated documentation (changelog).

## Additional information

Rainy should know the issue.
I've tested my edit already.

### CODE REVIEW

**Comment by ThePedroo** [comment]
We can`t accept that as it doesn`t conform [PerformanC JS syntax guidelines](https://github.com/PerformanC/contributing?tab=readme-ov-file#javascript--typescript-nodejs)

**Review by RainyXeon — CHANGES_REQUESTED** [review]
Please make it follows perfc js syntax rules before merging 
[Link here if you want to](https://github.com/PerformanC/contributing?tab=readme-ov-file#javascript--typescript-nodejs)

**Comment by SheepChef** [comment]
Removed conflicting code format.
With only two lines added to fix the code.

**Review by RainyXeon — COMMENTED** [review]
LGTM wait for Pedro

**Review by RainyXeon — APPROVED** [review]

**Comment by SheepChef** [comment]
image
Made the text in small card align with big card.


image
Improve the design of Header, added a divide line and align the text with the content underneath.

**Comment by SheepChef** [comment]
Ready to merge.
Moved module_amount up.

File: webroot/js/main.js [hunk 1]
```diff
@@ -107,7 +108,8 @@ export function setErrorData(errorLog) {
   const ptrace32Cmd = await exec('/data/adb/modules/zygisksu/bin/zygisk-ptrace32 info')
 
   if (ptrace32Cmd.errno === 0) {
-    const [ version_line, _, process_line, daemon_running_or_root_impl_line, daemon_running_line ] = ptrace32Cmd.stdout.split('\n')
+    const [ version_line, _, process_line, daemon_running_or_root_impl_line, daemon_running_line, module_amount_line] = ptrace32Cmd.stdout.split('\n')
```
Comment (ThePedroo) [inline webroot/js/main.js:111]:
```suggestion
    const [ version_line, _, process_line, daemon_running_or_root_impl_line, daemon_running_line, module_amount_line ] = ptrace32Cmd.stdout.split('\n')
```

File: webroot/js/main.js [hunk 2]
```diff
@@ -82,7 +83,7 @@ export function setErrorData(errorLog) {
       zygote64_status = UNEXPECTED_FAIL
     }
 
-    let modules_amount = lines[5].split(': ')[1]
+    let lines = ptrace64Cmd.stdout.split('\n')
```
Comment (ThePedroo) [inline webroot/js/main.js:86]:
```suggestion
    const lines = ptrace64Cmd.stdout.split('\n')
```

File: webroot/js/main.js [hunk 3]
```diff
@@ -62,7 +62,8 @@ export function setErrorData(errorLog) {
   const ptrace64Cmd = await exec('/data/adb/modules/zygisksu/bin/zygisk-ptrace64 info')
 
   if (ptrace64Cmd.errno === 0) {
-    const [ version_line, _, process_line, daemon_running_or_root_impl_line, daemon_running_line ] = ptrace64Cmd.stdout.split('\n')
+    const [ version_line, _, process_line, daemon_running_or_root_impl_line, daemon_running_line, module_amount_line] = ptrace64Cmd.stdout.split('\n')
```
Comment (ThePedroo) [inline webroot/js/main.js:65]:
```suggestion
    const [ version_line, _, process_line, daemon_running_or_root_impl_line, daemon_running_line, module_amount_line ] = ptrace64Cmd.stdout.split('\n')
```

File: webroot/js/main.js [hunk 4]
```diff
@@ -127,7 +129,7 @@ export function setErrorData(errorLog) {
       zygote32_status = UNEXPECTED_FAIL
     }
 
-    let modules_amount = lines[5].split(': ')[1]
+    let lines = ptrace32Cmd.stdout.split('\n')
```
Comment (ThePedroo) [inline webroot/js/main.js:132]:
```suggestion
    const lines = ptrace32Cmd.stdout.split('\n')
```

**Review by ThePedroo — COMMENTED** [review]
Just syntax nitpicks and it LGTM, wait for Rainy

**Comment by SheepChef** [comment]
Syntax optimized as you suggested.

**Comment by ThePedroo** [comment]
You forgot ptrace64 array one

**Pull request merged**